scripts/test-conversation.py

```diff
@@ -116,6 +120,9 @@ def main():
                     ciphers = [getattr(CipherSuite, arg)]
                 except AttributeError:
                     ciphers = [int(arg)]
+        elif opt == '-g':
```

Comment:
I think it would be nice to `try:` -> `except:` the `getattr()` so that if misspelled or unsupported group is provided the user is made aware of that.